### ISSUE TITLE
[BUG] Maximum call stack size exceeded when generating migrations 

### ISSUE THREAD

**Issue by EvertonWingert** [TiagoSilvaPereira/vemto-issues#35]
```
RangeError: Maximum call stack size exceeded
    at o.getFilePathByKey (app://./js/chunk-vendors.ca8bc0b1.js:37:32047)
    at o.get (app://./js/chunk-vendors.ca8bc0b1.js:37:31142)
    at s.get (app://./js/chunk-vendors.ca8bc0b1.js:115:2735)
    at e.exports.getTableData (app://./js/chunk-vendors.ca8bc0b1.js:389:4986)
    at e.exports.getItemIndex (app://./js/chunk-vendors.ca8bc0b1.js:389:6782)
    at e.exports.execute (app://./js/chunk-vendors.ca8bc0b1.js:89:1521)
    at h.executeRelationship (app://./js/chunk-vendors.ca8bc0b1.js:57:4571)
    at Object.__get [as get] (app://./js/chunk-vendors.ca8bc0b1.js:57:2312)
    at Proxy.belongsToAnotherEntity (app://./js/chunk-common.235694ae.js:1:53408)
    at app://./js/chunk-common.235694ae.js:1:44118
```

---

vemto 0.5.7
Electron 8.5.2
win32 10.0.19041
Locale: pt-BR

**Comment by TiagoSilvaPereira** [comment]
Hi @EvertonWingert thank you for reporting this problem. Could you please provide a screenshot of your Schema, so I can reproduce the error? Have a nice day, thank you

**Comment by Max2131970** [comment]
The same error

![image](https://user-images.githubusercontent.com/74265295/98717583-8c87c100-238d-11eb-96c8-a4f03c8b4c5b.png)

**Comment by TiagoSilvaPereira** [comment]
Hi @EvertonWingert it was fixed on **Vemto 0.5.9**. Cheers
